lexfloatclient.go

```diff
@@ -322,6 +322,22 @@ func HasFloatingLicense() int {
 	return int(status)
 }
 
+/*
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
+
+    RETURN CODES: LF_OK, LF_FAIL, LF_E_PRODUCT_ID, LF_E_LICENSE_EXISTS, LF_E_HOST_URL,
```

Comment:
@muneebkq Parameter is missing in the docs.

Reply:
added now